tests/common/fixtures/duthost_utils.py

```diff
@@ -598,3 +602,131 @@ def check_bgp_router_id(duthost, mgFacts):
             return False
     except Exception as e:
         logger.error("Error loading BGP routerID - {}".format(e))
+
+
+@pytest.fixture(scope="module")
+def convert_and_restore_config_db_to_ipv6_only(duthosts):
+    """Back up the existing config_db.json file and restore it once the test ends.
+
+    Some cases will update the running config during the test and save the config
+    to be recovered after reboot. In such a case we need to backup config_db.json before
+    the test starts and then restore it after the test ends.
+    """
+    config_db_file = "/etc/sonic/config_db.json"
+    config_db_bak_file = "/etc/sonic/config_db.json.before_ipv6_only"
+
+    # Sample MGMT_INTERFACE:
+    #     "MGMT_INTERFACE": {
+    #         "eth0|10.212.69.117/27": {
+    #             "forced_mgmt_routes": [
+    #                 "10.212.69.2"
+    #             ],
+    #             "gwaddr": "10.212.69.97"
+    #         },
+    #         "eth0|2603:10e2:140:3000::96/122": {
```

Comment:
Please use fake IP here. Both IPv4 and IPv6.

Reply:
Refined